projects.md

```diff
@@ -0,0 +1,160 @@
+
+The Monitored data includes geolocation mapping on a web interface with real-time information reported from nodes, and plotting logical mesh topology on a geographical map. This will be packaged in a way that makes it very easy for other mesh networks to adapt to their own monitoring needs.
+
+**Status:** Concept  
+Lead required to further develop proof-of-concept work.
+
+**Skills**  
+Devops, Terraform, Docker, devops, Prometheus, mapping, data visualization, networking, web design, Google Maps, Open Street Maps, shell scripts, python, cjdns.
+
+[Monitoring GitHub page](https://github.com/tomeshnet/monitoring)
+
+---
+Collaboration and Outreach Tools
+
+---
+
+### Toronto Mesh Website
+
+The primary digital space for people to learn about Toronto Mesh. It must represent the current focus of our organization and remain accessible to diverse audiences.
+
+**Status:** Production (live)  
+Need to be spun up with provisioning tools and served over cjdns, IPFS, DAT, etc.
+
+**Skills**  
+Web design, user research, javascript, Terraform, peer-to-peer hosting.
+
+[Toronto Mesh Website GitHub page](https://github.com/tomeshnet/tomesh.net)
+
+### Meshbot
+
+The chat bot our Matrix chat uses to send event reminders and answer basic questions about the mesh.
+
+**Status:** Production
+
+**Skills**  
+Programming, user research, technical documentation
+
+[Meshbot GitHub page](https://github.com/tomeshnet/meshbot)
+
+### Mesh Community Assets
+
+Our tools and infrastructure, such as the Matrix chat and website, are available for other groups to adapt and easily deploy their own mesh network.
+
+**Status:** Production  
+**Need experienced devops lead.**
+
+**Skills**  
+Terraform, Docker, devops.
+
+Visit the [Tools Transfer Guide](https://github.com/tomeshnet/documents/blob/master/governance/tools-transfer.md) for an overview of our community assets.
+
+### IPFS Live Streaming
+
+A  live streaming solution built with IPFS that uses [Terraform](https://www.terraform.io/) to spin up a cluster on [Digital Ocean](https://www.digitalocean.com/).
+
+Work on this project began with Toronto Mesh members experimenting with HTTP Live Streaming on the IPFS network. This evolved into a production setup in order to support the live stream at [Our Networks 2018](https://ournetworks.ca/). The repository now contains documentation and all the necessary software to live stream a small conference, complete with authenticated publishing to an RTMP endpoint and embeddable video players tested on desktop and mobile browsers.
+
+**Status:** Production
+This solution was used at Our Networks 2018.
+
+**Skills**  
+Live video streaming, ffmpeg, Terraform, shell scripts, devops, web design.
+
+[IPFS Live Streaming GitHub page](https://github.com/tomeshnet/ipfs-live-streaming)
+
+---
+Community Networks and Outreach
+
+---
+
+### Toronto Meshnet
+
+All deployment efforts and governance for a physical mesh network in Toronto.
```

Comment:
I think this could be expanded with some of the language we've used elsewhere to better describe what the intent of this project is.